TESTING_INSTRUCTIONS.md

```
@@ -0,0 +1,165 @@
+            ```
+            (Add this to your `~/.bashrc`, `~/.zshrc`, or current session)
+        *   **Windows (Command Prompt):**
+            ```cmd
+            set GOOGLE_API_KEY=YOUR_API_KEY_HERE
+            ```
+        *   **Windows (PowerShell):**
+            ```powershell
+            $env:GOOGLE_API_KEY="YOUR_API_KEY_HERE"
+            ```
+*   **Optional: Google GenAI Model Name (`GOOGLE_GENAI_MODEL`):**
+    *   The `server.py` defaults to `models/gemini-1.5-flash-latest` or a model specified by `GOOGLE_GENAI_LIVE_MODEL`. If you want to use a different compatible model, you can set the `GOOGLE_GENAI_MODEL` environment variable similarly to the API key.
```

markdownlint-cli2 (0.17.2)

Unordered list indentation
Expected: 2; Actual: 4

(MD007, ul-indent)